sdk/notificationhubs/notification-hubs/samples-dev/deno/sendDirectNotification.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how the sendDirectNotification() method can be used to send a direct
+ * notification using APNs.  This sends a JSON message to an APNs given device token and returns
+ * a Tracking ID which can be used for troubleshooting with the Azure Notification Hubs team.
+ *
+ * See https://docs.microsoft.com/rest/api/notificationhubs/direct-send
+ * to learn about Direct Send.
+ *
+ *
+ * @summary Demonstrates how to send direct notifications using Azure Notification Hubs
+ * @azsdk-weight 100
+ */
+
+import * as dotenv from "npm:dotenv@16.0.3";
+import * as process from "https://deno.land/std@0.104.0/node/process.ts";
```

Comment:
could [Deno APIs](https://deno.land/manual@v1.27.2/basics/env_variables#env-file) be used to load those env vars?